### DISCUSSION TITLE
/var/{export, importexport} directories content

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1706] [Ideas]
Magento doesn't have an option in Backend to delete the content for the following directories:

/var/export
var/importexport

Most of the bots are trying to access these directories. Maybe we should find a way to clean up the content periodically (cron, manually, ...).

**Comment by fballiano** [comment]
I guess I could add it to my imagecleaner module :-D

**Reply by fballiano** [reply]
@ADDISON74 I've added this feature in the 0.9.0 version of my imagecleaner ;-)

**Reply by addison74** [reply]
You know I fully appreciate your work but I don't think it's the best option, because it has nothing to do with your extension (which btw is the best for orphan images). You should create a PR in OpenMage and add the feature in a section (e.g. Cache Management). As a bonus I would add the possibility to run or not a configurable cronjob at a specified time. Thus, you either do the operation manually or you determine when to delete these folders periodically.

**Comment by fballiano** [comment]
since we added the flush media/import some time ago I think it can be
suited for that kind of purpose too.

I'll probably also add a cron based flushing sometime in the future.

Il Mar 13 Lug 2021, 16:14 ADDISON ***@***.***> ha scritto:

**Reply by addison74** [reply]
In those directories we don't find images but only CSV files. Let's say it's a temporary bonus until the feature is added to OpenMage. You can propose it as a PR and later someone can add a cronjob feature.

**Reply by fballiano** [reply]
I know that, I can't see any reason why that module can't be more open. while OM never had those things in the first place. Also, I won't add a PR that most probably will lay there forever without getting approved by anyone.

**Reply by addison74** [reply]
There are PRs that are really complex, but also some lighter ones that add visual features and I think they can be shared by many.

Do not start from the idea that your work will not be appreciated. Look how together we made your extension a version that no ImaClean user has ever dreamed of. As long as I'm here I'll give you feedback on whatever you do. At least I hope so.
